recipes/libenzyme/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "libenzyme" %}
+{% set version = "0.0.23" %}
+{% set build_number = 0 %}
+
+# Needed for the linter even though it's specified in conda_build_config.yaml
+{% set llvm_major_version = llvm_major_version or "11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wsmoses/Enzyme/archive/v{{ version }}.tar.gz
+  sha256: ce15ce1b839ba68e0941ce39730ab4df938eb1a6383feaffbccd3276fac22d14
+
+build:
+  number: {{ build_number }}
+  string: llvm{{ llvm_major_version }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  # Loadable modules are not supported by LLVM on Windows
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - {{ "libllvm" + llvm_major_version }}
+    - llvmdev {{ llvm_major_version }}
```

Comment:
It's likely worth adding clangdev here so the nicer helper ClangEnzyme-XX.so gets built too.